spec/dummy/spec/features/integration_spec.rb

```diff
@@ -176,6 +176,15 @@ def change_text_expect_dom_selector(dom_selector)
   end
 end
 
+feature "renderedHtml from generator function", :js do
+  subject { page }
```

Comment:
Do you prefer to call it `subject` instead of `page`? To me, page is more explicit and specific in this context.